src/webgpu/api/validation/createTexture.spec.ts

```diff
@@ -289,19 +289,43 @@ g.test('it_is_invalid_to_submit_a_destroyed_texture_before_and_after_encode')
     }, !_success);
   });
 
-g.test('it_is_invalid_to_have_an_output_attachment_texture_with_non_renderable_format')
-  .params(poptions('format', kAllTextureFormats))
+g.test('texture_usage')
+  .desc(
+    `Test texture usage (single usage or combined usages) for every texture formats and every dimension types`
```

Comment:
I believe there are also restrictions based on the sample count, can we add 1 and 4 sample counts?

Reply:
Good point. I have tested multisample and `storage` usage conflict at sampleCount: https://github.com/gpuweb/cts/pull/478/files#diff-e0bc33c614ca9f7cdbbf7fdec58243d4af5681749e8d56739c95e970e521d15fR339. And the current tests are really a lot. So I didn't test that again. Otherwise, it will double the amount of test. Is this OK?